applications/tasks/src/modules/kt_numbers.erl

```diff
@@ -259,6 +330,30 @@ account_id(_) -> 'false'.
 carrier_module(Data) ->
     lists:member(Data, knm_carriers:all_modules()).
 
+-spec ported_in(ne_binary()) -> boolean().
+ported_in(Cell) -> is_cell_boolean(Cell).
+
+-spec 'cnam.inbound'(ne_binary()) -> boolean().
+'cnam.inbound'(Cell) -> is_cell_boolean(Cell).
+
+-spec 'prepend.enabled'(ne_binary()) -> boolean().
```

Comment:
Typically functions that return booleans are named to suggest that. `is_prepend_enabled` would be better IMO. `is_cnam_inbound_enabled` above. For someone that doesn't like ticky-ing atoms, its an odd choice to create function names that would require the tickies! ;)